README.md

```diff
@@ -208,6 +208,18 @@ source activate nums
 pip install -e ".[testing]"
 ```
 
+For Windows, follow the steps below if you do not have Anaconda already downloaded. 
+1. Follow the steps to download the Anaconda Installer. https://docs.anaconda.com/anaconda/install/windows/
+2. During the setup, select that you want Anaconda to be added to the PATH variable. 
+3. Fork/Clone the nums-project repository
+4. Open Command Prompt and follow the instructions below:
```

Comment:
Can you update the README so that the #Installation section has two subsections: One for Linux/Mac and one for Windows, and copy and adapt these instructions for that section? It's okay if there are redundant instructions between the two subsections.

Can you update these instructions to read as follows:
**For Windows, we recommend Anaconda. Download and install Anaconda for Windows [here](https://docs.anaconda.com/anaconda/install/windows/). Make sure to add Anaconda to the PATH variable during installation.**